src/shared/format-table.c

```diff
@@ -380,6 +380,20 @@ int table_add_cell_full(
         return 0;
 }
 
+int table_add_string_cell(Table *t, TableCell **ret_cell, const char *format, ...) {
```

Comment:
i think this should have a name suffixed with "f", given that this is really about format strings: mayve `int table_add_cell_stringf(…)`?

We usually suffix with "f" for format strings taking inspiration from printf()... (but there are admittedly exceptions already)